### PR TITLE
65 create session endpoint

### PR DESCRIPTION
**Pull request by jjrobertson14**

### CODE REVIEW

File: server/api.py [hunk 1]
```diff
@@ -249,24 +249,25 @@ def delete_map(claims, map_id):
             return internal_error()
         return jsonify(success=map_id), 200, json_tag
 
-    def create_session(claims):
+    def create_session(claims,token_user):
```
Comment (joshherkness) [inline server/api.py]: Add a space before "token_user"

File: server/server.py [hunk 2]
```diff
@@ -113,6 +113,14 @@ def create_session(claims):
     """ Creates a session with the given map_id and token user's id """
     return Api.create_session(claims)
 
+@api.route('/sessions/', methods=['POST'])
+@protected
+@expiration_check
+def create_session(claims,token_user):
```
Comment (joshherkness) [inline server/server.py]: Add a space before "token_user"

File: server/server.py [hunk 3]
```diff
@@ -113,6 +113,14 @@ def create_session(claims):
...
+    """ Creates a session with the given map_id and token user's id """
+    return Api.create_session(claims,token_user)
```
Comment (joshherkness) [inline server/server.py]: Add a space before "token_user"

**Review by AnthonyHewins — APPROVED** [review]

**Comment by AnthonyHewins** [comment]
I'm going to pull these in right now because some of the things i have to do are waiting on this being done. if there's an issue we will just have to fix it later

**Pull request merged**